### PR TITLE
create menu screen + change logic of walletaddress button with share …

### PR DESCRIPTION
**Pull request by oasm1982**
step 1
step 2
step 3

### CODE REVIEW

**Comment by oasm1982** [comment]
step 4

**Comment by oasm1982** [comment]
WhatsApp Image 2022-11-23 at 16 02 29
WhatsApp Image 2022-11-23 at 16 02 32

File: components/atoms/TokenIcon.tsx [hunk 1]
```diff
@@ -4,26 +4,29 @@ import { Image } from "./Image";
...
 import { dictionaryTokenIcon } from "../../constants/tokenIcon";
 import { CHAIN_IDS } from "../../constants/chains";
+import { useProtocol } from "../../constants/protocol";
+
 
-// in 
 interface TokenIconInterface {
     address: string // the address of the asset
     size: number // the size of the icon asset
     chainId: number
```
Comment (samhv) [inline components/atoms/TokenIcon.tsx]: given you removed `chainId` from the component's props, let's remove it from the interface too

File: components/screens/MenuScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                <View alignItems="center" justifyContent="center">
+                    <Text style={styles.thanksText}>
+                             Built with ♥️ from Aave Comunity.
+                    </Text> 
+                </View>    
+            </View>
+    );
+}
+
+export const SignedoutButton = () => {
```
Comment (samhv) [inline components/screens/MenuScreen.tsx]: typo -- `SignedOutButton`

**Review by samhv — CHANGES_REQUESTED** [review]
couple small comments
Please remove `assets/.DS_Store`

**Review by samhv — APPROVED** [review]
lgtm

**Pull request merged**